pkg/ingester/client/timeseries.go

```diff
@@ -15,6 +15,7 @@ var (
 	expectedTimeseries       = 100
 	expectedLabels           = 20
 	expectedSamplesPerSeries = 10
+	expectedMetadata         = 200
```

Comment:
On a freshly-minted Prometheus, monitoring itself (on my laptop) I have about ~160 metadata. 200 seems like a good enough number given the current approach, if they were sent together this should probably change to =< expected time series. I'm happy to add a comment if it makes sense.